### DISCUSSION TITLE
GLideN64 homepage. Feedback needed

### DISCUSSION THREAD

**Discussion by gonetz** [gonetz/GLideN64#2335] [General]
GLideN64 project never had its own home page. There are several separated resources related to it. The most important is GitHub, but it is for development and for developers. Patreon for news and support. My blog on Blogspot for articles and most important news.

GLideN64 home page on EmuNations can be considered as a starting point, where common users may find all necessary resources such as news, status, downloads, compatibility list and also links to other resources. All in one place. The page is already available: https://www.emunations.com/gliden64

Content, layout, design are subjects to discuss. I need your feedback about it. I myself understand nothing in design and usability.

**Comment by ghost** [comment]
The download section links to the latest artifact built by appveyor. Instead, it should link to the latest artifact from the master branch to prevent artifact that come from incomplete pull requests to be downloaded as if they were the latest development release.
I believe the correct link is
https://ci.appveyor.com/project/gonetz/gliden64/build/artifacts?branch=master
It should be similar in travis too.

**Comment by gonetz** [comment]
AppVeyor link is fixed, thanks!
As for Travis, I don't see any links to results of the build. I'm not sure that Travis provides such a service.

**Comment by Papermanzero** [comment]
Why not making a github page? There you can link to all relevant resources. It can act as hub for patreon, appvoyer, github, etc.

**Comment by gonetz** [comment]
I can't. I have no such skills.

**Comment by Papermanzero** [comment]
Do you have requirements for the webpage?
Wireframe etc.? Shall it be a static page or should it be dynamic? I guess you do not need any database in the background. So a static page should be sufficient. Except you want to release your builds on the page. We  can use the emunation page as starting point but it is more or less a list of iframes (references) to other pages.

I can try to support you and program a little bit your page. I can do it either in php or pure html/css.

**Comment by gonetz** [comment]
I personally need no webpage. GitHub is all I need as a developer.
WebPage is for users. What I expect to see as a user when I open a page of a project which I'm not familiar with? Some general info, recent news, hardware requirements, download links, compatibility list with current status, best settings and tips how to bypass existing issues. The page on EmuNations is close to what I expect from emulation project home page.
